packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -133,7 +133,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
             const annotationCategories = await tspClient.fetchAnnotationsCategories(this._selectedExperiment?.UUID, output.id);
             const annotationCategoriesResponse = annotationCategories.getModel();
             if (annotationCategories.isOk() && annotationCategoriesResponse) {
-                const categoriesList = annotationCategoriesResponse.model.annotationCategories;
+                const categoriesList = annotationCategoriesResponse.model?.annotationCategories;
```

Comment:
It still causes an exception later at reception of the signal.

I think we still need to fire the signal to let the users know there are no categories, either by replacing with empty array or having the signal parameter possibly undefined.

Reply:
Yeah, I was doing that before, ESLint told me to do this instead, Thanks!

Reply:
Please check the updated version of the patch.